packages/bbui/src/Modal/Modal.svelte

```diff
@@ -53,6 +54,9 @@
   }
 
   async function focusModal(node) {
+    if (!autoFocus) {
+      return
+    }
```

Comment:
Need to add a prop for this because when you have a very long modal this auto focus behaviour scrolls right to the bottom of the modal which is jarring.